src/libraries/System.Runtime.InteropServices/tests/ComInterfaceGenerator.Tests/RcwAroundCcwTests.cs

```diff
@@ -0,0 +1,379 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.Marshalling;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using SharedTypes.ComInterfaces;
+using Xunit;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using static ComInterfaceGenerator.Tests.ComInterfaces;
+
+namespace ComInterfaceGenerator.Tests
+{
+    public partial class RcwAroundCcwTests
+    {
+        static TInterface CreateWrapper<TClass, TInterface>() where TClass : TInterface, new()
+        {
+            var cw = new StrategyBasedComWrappers();
+            var comPtr = cw.GetOrCreateComInterfaceForObject(new TClass(), CreateComInterfaceFlags.None);
+            var comObject = cw.GetOrCreateObjectForComInstance(comPtr, CreateObjectFlags.None);
+            var ifaceObject = (TInterface)comObject;
+            return ifaceObject;
+        }
+
+        [Fact]
+        public void IGetAndSetInt()
+        {
+            var obj = CreateWrapper<GetAndSetInt, IGetAndSetInt>();
+            obj.SetInt(0);
+            _ = obj.GetInt();
```

Comment:
Should this validate what `GetInt` returns? And maybe set to something that is not the default value?